tests/examples/source/devfiles/nodejs/devfile-with-multiple-containers.yaml

```diff
@@ -0,0 +1,53 @@
+schemaVersion: 2.0.0
+metadata:
+  name: nodejs
+starterProjects:
+  - name: nodejs-starter
+    git:
+      remotes:
+        origin: "https://github.com/odo-devfiles/nodejs-ex.git"
+components:
+  - name: runtime
+    container:
+      image: registry.access.redhat.com/ubi8/nodejs-12:1-36
+      memoryLimit: 1024Mi
+      endpoints:
+        - name: "3000-tcp"
+          targetPort: 3000
+      mountSources: true
+  - name: funtime
+    container:
+      image: registry.access.redhat.com/ubi8/nodejs-12:1-36
+      memoryLimit: 1024Mi
+      mountSources: true
```

Comment:
I am facing `Back-off restarting failed container` with this devfile. This might be cause behind the deployment failures in the PR. We can use the springboot devfile instead.

Reply:
Switched to springboot devfile